packages/universal-components/src/RangeDatePicker/components/DayItemArrow.web.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { StyleSheet, type StylePropType } from '../../PlatformStyleSheet';
+import { ExtendedTouchable } from '../../ExtendedTouchable';
+import { SimpleArrow } from './SimpleArrow';
+
+export type Props = {|
+  +style?: StylePropType,
+  +onPress?: () => void,
```

Comment:
The onPress should be required, I suppose

Reply:
Actually I'm not sure about that. I'm going to remove this `onPress` when I will implement gestures on Android. Then it would be duplicity, because you could grab invisible `DraggableItem` or click on `DayItemArrow` which is rendered behind the `DraggableItem`.

Reply:
OK make sense to me